Source/CTPanoramaView.swift

```diff
@@ -203,6 +204,9 @@ fileprivate extension CMDeviceMotion {
         sceneView.backgroundColor = UIColor.black
         sceneView.delegate = self
         sceneView.isPlaying = true
+        if #available(iOS 9.0, *) {
+            _ = sceneView.audioEngine
```

Comment:
Do you know what this is doing / why this works? I'm guessing that's a lazy property, but do we know why it crashes without this?

Also, is there any reason to support iOS 8 in this pod?